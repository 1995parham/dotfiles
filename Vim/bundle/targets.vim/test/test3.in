A
a ( b ) c

B ( a ) b
a ( b ) C

D ( a
b ) c

a ( b
c ) E

a ( b
F ( ) c
d ) e

a ( b
c ( ) G
d ) e

a ( b ) H ( c ) d

a ( b ) c
I " this must be the last pair test

J
a 'b' c

K 'a' b
a 'b' L

a ' b
M ' c
d ' e

a ' b
c ' N
d ' e

a ' b ' c
O " this must be the last quote test

