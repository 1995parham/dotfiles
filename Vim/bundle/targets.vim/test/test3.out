A
a (A) c

B (B) b
a (C) C

D (D) c

a (E) E

a (F) e

a (G) e

a ( b ) H (H) d

a (I) c
I " this must be the last pair test

J
a 'J' c

K 'K' b
a 'L' L

a 'M' c
d ' e

a ' b
c 'N' e

a 'O' c
O " this must be the last quote test

